src/code/z_sub_s.c

```diff
@@ -987,4 +1110,54 @@ s32 func_8013E8F8(Actor* actor, GlobalContext* globalCtx, f32 xzRange, f32 yRang
     return func_8013E748(actor, globalCtx, xzRange, yRange, exhangeItemId, &yawTols, SubS_ActorAndPlayerFaceEachOther);
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E950.s")
+/**
+ * Computes the necessary HeadRot and TorsoRot adjustments to smoothly turn an actors's head and torso to different yaw
+ * and pitch target points
+ *
+ * @param[in] worldPos the actor's world position
+ * @param[in] focusPos the actor's focus position
+ * @param[in] shapeYRot the actor's shape's Y rotation
+ * @param[in] yawTarget the target point to determine desired yaw
+ * @param[in] pitchTarget the target point to determine desired pitch
+ * @param[in,out] headZRotAdj the computed actors' head's Z rotation
+ * @param[in,out] headXRotAdj the computed actors' head's X rotation
+ * @param[in,out] torsoZRotAdj the computed actors' torso's Z rotation
+ * @param[in,out] torsoXRotAdj the computed actors' torso's X rotation
+ * @param[in] headZRotAdjMax the max head's Z rotation
+ * @param[in] headXRotAdjMax the max head's X rotation
+ * @param[in] torsoZRotAdjMax the max torso's Z rotation
+ * @param[in] torsoXRotAdjMax the max torso's X rotation
+ */
+s32 SubS_TurnToPointMultiTarget(Vec3f* worldPos, Vec3f* focusPos, s16 shapeYRot, Vec3f* yawTarget, Vec3f* pitchTarget,
+                                s16* headZRotAdj, s16* headXRotAdj, s16* torsoZRotAdj, s16* torsoXRotAdj,
+                                u16 headZRotAdjMax, u16 headXRotAdjMax, u16 torsoZRotAdjMax, u16 torsoXRotAdjMax) {
+    s16 yaw;
+    s16 pad;
+    s16 pad2;
```

Comment:
I assume there's no choice for the padding here? Also can be defined where declared.

Reply:
From what I've seen, padding has to be two separate `s16` pads. Array/`s32` does not work.